pubspec.yaml

```
@@ -5,7 +5,7 @@ documentation: https://github.com/Workiva/contextual_message
 homepage: https://github.com/Workiva/contextual_message
 
 environment:
-  sdk: '>=2.11.0 <3.0.0'
+  sdk: '>=2.12.0 <3.0.0'
 
 dev_dependencies:
   dart_dev: '>=3.9.0 <5.0.0'
```

```suggestion
  dart_dev: ^4.0.0
```
Let's ensure we resolve to the null safe version of dart_dev